Assets/Scripts/Game/UserInterface/DaggerfallFont.cs

```diff
@@ -134,19 +139,6 @@ public DaggerfallFont(string arena2Path, FontName font = FontName.FONT0003)
 
         #region Glyph Rendering
 
-        public int GetSDFGlyphWidth(int ascii)
```

Comment:
Please don't remove this. It's part of the DaggerfallFont feature set. Just because nothing needs it right now doesn't mean that it shouldn't be available. :)

Reply:
ah sry, thought I added this one, since I wrote several similar helper functions when I first tried to use old Texture2D approach

Reply:
All good! I didn't have it in the right spot anyway, so it looked out of place. :)